application/config/config.go

```diff
@@ -492,18 +492,18 @@ func (c *Config) ConfigureLogging(server lsp.Server) {
 
 	logLevel, err = zerolog.ParseLevel(c.LogLevel())
 	if err != nil {
-		_, _ = fmt.Fprintln(os.Stderr, "Can't set log level from flag. Setting to default (=info)")
```

Comment:
This will most likely triggering an inspection / warning in Goland.

Reply:
I don't use Goland. I'm satisfying the linter only. There is a rule which requires checked errors, but I've disabled it for the time being (it'd bring too many changes into this PR).

Reply:
I can revert this, but it's going to be awkward to maintain conventions that only some of the IDEs care about. I'd prefer that we standardize on golangci-lint. Maybe I can find a way to catch this with a lint rule?

Reply:
I agree, we can disable that inspection in goland and rely on the linter.